test/cctest/testapi.cpp

```diff
@@ -344,6 +344,18 @@ TEST(ValueRef, Basic2) {
     EXPECT_EQ(ii, 123);
 }
 
+TEST(ValueRef, Basic3) {
+    Evaluator::execute(g_context.get(), [](ExecutionStateRef* state) -> ValueRef* {
+        auto value = ValueRef::create(-1);
```

Comment:
It would be nice if you add more tests using variety values!

Reply:
Thanks for the review. I added one more test which checks the maximum value of uint32_t.